pkg/proxy/serviceport.go

```diff
@@ -201,13 +193,6 @@ func newBaseServiceInfo(service *v1.Service, ipFamily v1.IPFamily, port *v1.Serv
 		internalPolicyLocal: internalPolicyLocal,
 	}
 
-	// v1.DeprecatedAnnotationTopologyAwareHints has precedence over v1.AnnotationTopologyMode.
-	var exists bool
-	info.hintsAnnotation, exists = service.Annotations[v1.DeprecatedAnnotationTopologyAwareHints]
-	if !exists {
-		info.hintsAnnotation = service.Annotations[v1.AnnotationTopologyMode]
```

Comment:
Oh, we should update the documentation for these two annotations too (`staging/src/k8s.io/api/core/v1/annotation_key_constants.go`), but please do that in a separate PR because that will need an API approver.

Reply:
Ack. Yes will raise a separate PR for starting their deprecation.